website/src/en/playground.html

```diff
@@ -270,7 +270,7 @@ <h2 class="tagline"><em>Opinionated</em> code formatter</h2>
     </p>
   </footer>
 
-  <script src="lib/sw-toolbox-companion.js" data-service-worker="service-worker.js"></script>
+  <script src="lib/sw-toolbox-companion.js" data-service-worker="lib/service-worker.js"></script>
```

Comment:
Moving service worker to a sub directory will change its scope. Is this intentional?